### PR TITLE
Added New Member Profile.md to ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by Elijah699**
Signed-off-by: Elijah Edun <edunelijah18@gmail.com>

**Description**

This PR fixes #1325

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Thanks for this, @Elijah699. I tossed in a small update.

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,3 +1,4 @@
 {
-    "git.alwaysSignOff": true
+    "git.alwaysSignOff": true,
```
Comment (leecalcote) [inline .vscode/settings.json:2]: @Elijah699 will you kindly remove this file? We can skip committing it in this PR.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
@Elijah699 please populate the issue description with the issue number that this PR fixes.

**Review by Ruth-ikegah — CHANGES_REQUESTED** [review]
Hi @Elijah699 thanks for the PR, @leecalcote added some changes. Please add that. Also, fill in the description! Thanks!

File: .vscode/settings.json (hunk 1 above)
Comment (Elijah699) [reply .vscode/settings.json:2]: Yes, let me do that rightaway

**Comment by Elijah699** [comment]
Filled in issue number and removed  .vscode/settings.json file

**Comment by Ruth-ikegah** [comment]
Hi @Elijah699 nice work, please check the DCO check and amend your commits.

**Review by leecalcote — APPROVED** [review]
The updates that I made though GitHub’s web editor caused DCO check to fail. GitHub is terrible in this regard.

thanks for this update, @Elijah699 and review @Ruth-ikegah. Looks good. 👍

**Review by Ruth-ikegah — APPROVED** [review]
Looking good now, welldone @Elijah699

**Pull request merged**